canonical-websites/partners.ubuntu.com
# Change 'reseller' to 'channel partners' site wide

## Done

Changed instances of 'Reseller' to 'Channel Partners' site wide.
## QA
- Navigate to /programmes and check that middle box on bottom row says 'Channel Partners'
- Navigate to /find-a-partner and check that filter tickbox under 'Programme' has 'Channel Partners' as an option.
## Related

Trello: https://trello.com/c/dgCCASvV
